App.tsx

```
@@ -90,24 +90,24 @@ const BottomNav = ({ onNotesClick }: { onNotesClick: () => void }) => {
       <div className="flex items-center justify-around h-16 max-w-md mx-auto">
-          <Link to="/" className={getLinkClass(isProjects)}>
+          <Link to="/" className={getLinkClass(isProjects)} aria-current={isProjects ? 'page' : undefined}>
               <span className="material-symbols-outlined text-[24px]">grid_view</span>
               <span className="text-[10px] font-bold">Projects</span>
           </Link>
           <button
               onClick={onNotesClick}
-              className="flex flex-col items-center justify-center w-16 h-full gap-1 text-gray-500 dark:text-gray-400 hover:text-gray-900 dark:hover:text-white transition-colors"
+              className="flex flex-col items-center justify-center w-16 h-full gap-1 text-gray-500 dark:text-gray-400 hover:text-gray-900 dark:hover:text-white transition-colors focus-visible:ring-2 focus-visible:ring-primary focus-visible:outline-none rounded-xl active:scale-95"
               aria-label="Open Notes"
           >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)